brain/circuit.cpp

```diff
@@ -147,6 +150,26 @@ class Circuit::Impl
         return brion::Target::parse( _targetParsers, target );
     }
 
+    GIDSet getRandomGIDs( const float fraction,
+                          const std::string& target ) const
+    {
```

Comment:
You must provide a way to make the result deterministic. I'd suggest to either add an environment variable to choose the seed or always use the same seed. In particular, I think the C++11 code is using the most random device possible by default.